client/scss/components/_button.scss

```diff
@@ -341,6 +341,18 @@
   background-color: transparent;
 }
 
+.button-tooltip {
+  background: transparent;
+  border: 0;
+  padding: 0;
+  color: inherit;
+  text-align: left;
```

Comment:
would be good to avoid text alignment if possible - this may not be necessary and also makes it harder for RTL languages.